--- a/cmake/SearchForStuff.cmake
+++ b/cmake/SearchForStuff.cmake
@@ -108,7 +108,7 @@ disable_compiler_warnings_for_target(speex)
 
 # Find the Qt components that we need.
 if(ENABLE_QT_UI)
-	find_package(Qt6 6.10.0 COMPONENTS CoreTools Core GuiTools Gui WidgetsTools Widgets LinguistTools REQUIRED)
+	find_package(Qt6 6.8.0 COMPONENTS CoreTools Core GuiTools Gui WidgetsTools Widgets LinguistTools REQUIRED)
 
 	if (Qt6_VERSION VERSION_GREATER_EQUAL 6.10.0)
 		find_package(Qt6 COMPONENTS CorePrivate GuiPrivate WidgetsPrivate REQUIRED)
--- a/pcsx2-qt/GameList/GameListWidget.cpp
+++ b/pcsx2-qt/GameList/GameListWidget.cpp
@@ -64,6 +64,13 @@ static constexpr std::array<int, GameLis
 static_assert(static_cast<int>(DEFAULT_COLUMN_WIDTHS.size()) <= GameListModel::Column_Count,
 	"Game List: More default column widths than column types.");
 
+#if QT_VERSION < QT_VERSION_CHECK(6, 9, 0)
+	#define beginFilterChange()
+#endif
+#if QT_VERSION < QT_VERSION_CHECK(6, 10, 0)
+	#define endFilterChange(x) invalidateRowsFilter()
+#endif
+
 class GameListSortModel final : public QSortFilterProxyModel
 {
 public:
